src/Controller/Component/AuthComponent.php

```diff
@@ -597,7 +597,7 @@ public function deny($actions = null)
     public function setUser(array $user)
     {
         $this->session->renew();
-        $this->session->write($this->sessionKey, $user);
+        $this->session->write($this->sessionKey . '.user', $user);
```

Comment:
This needs to stay User. Repurposing the property will very likely break userland code too.